packages/react-form/src/hooks/list/list.ts

```diff
@@ -33,13 +33,13 @@ export interface FieldListConfig<Item extends object> {
  * In it's simplest form `useList` can be called with a single parameter with the list to derive default values and structure from.
  *
  * ```typescript
- * const field = useList([{title: '', description: ''}, {title: '', description: ''}]);
+ * const list = useList([{title: '', description: ''}, {title: '', description: ''}]);
  * ```
  *
  * You can also pass a more complex configuration object specifying a validation dictionary.
  *
  * ```tsx
- *const field = useField({
+ *const list = useList({
```

Comment:
The docs here had confusing variables that didn't translate to the code below it.